src/Microsoft.SqlTools.ResourceProvider.Core/AccountOptionsHelper.cs

```diff
@@ -0,0 +1,20 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+
+namespace Microsoft.SqlTools.ResourceProvider.Core
+{
+    internal class AccountOptionsHelper
+    {
+        /// <summary>
+        /// The name of the IsMsAccount option
+        /// </summary>
+        internal const string IsMsAccount = "IsMsAccount";
+        /// <summary>
+        /// The name of the Tenants option
+        /// </summary>
+        internal const string Tenants = "Tentants";
```

Comment:
Tenants is spelled wrong

Reply:
Hmmm... somehow this file wasn't used. Will delete.